src/jit/lclvars.cpp

```diff
@@ -1783,7 +1774,7 @@ bool Compiler::lvaCanPromoteStructType(CORINFO_CLASS_HANDLE typeHnd, lvaStructPr
             //
             if (pFieldInfo->fldSize < TARGET_POINTER_SIZE)
             {
-                structPromotionHelper->SetRequiresScratchVar();
+                SetRequiresScratchVar();
```

Comment:
Since now it's the helper that sets it I'd drop the function and set the data member directly